api/plugin.go

```diff
@@ -39,8 +29,8 @@ func (s *Server) SignPluginMessages(c echo.Context) error {
 	if len(req.Messages) != 1 {
 		return fmt.Errorf("plugin signing requires exactly one message hash, current: %d", len(req.Messages))
 	}
-	if len(req.Transactions) != 1 {
-		return fmt.Errorf("plugin signing requires exactly one transaction, current: %d", len(req.Transactions))
+	if len(req.Transaction) != 1 {
+		return fmt.Errorf("plugin signing requires exactly one transaction, current: %d", len(req.Transaction))
```

Comment:
This is not required anymore since Transactions isn't an array now - it's just checking the string length, which is somewhat pointless